packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -322,6 +328,10 @@ const DropDownEditor = TextBox.inherit({
         this.callBase();
     },
 
+    _setFieldAriaLabel() {
+        this.setAria('labelledby', TEMPLATE_CONTENT_LABEL);
```

Comment:
The labelledby attribute should be linked with the ID of the label. Please check Artem's [sample](https://stackblitz.com/edit/vitejs-vite-cw79as?file=package.json) and see how it should work on SelectBox without fieldTemplate.